modules/homa-lite/src/lib.rs

```diff
@@ -868,6 +912,18 @@ pub mod module {
 				Self::xcm_dest_weight(),
 			)
 		}
+
+		/// Helper function that update the storage with the new
+		fn update_total_staking_currency_storage(new_total: Balance) -> DispatchResult {
```

Comment:
most of the time we are modify the current value, so could make this takes `f: impl FnOnce(Balance) -> Balance` and `*current = f(current)`

Reply:
Updated function and funtion signature